eslint-plugin-skyscanner-no-axios/src/rules/no-axios.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  create: (context) => ({
+    CallExpression: (node) => {
+      if (
+        node.callee.name === 'require' &&
+        node.arguments.length > 0 &&
+        typeof node.arguments[0].value === 'string' &&
+        (node.arguments[0].value === 'axios' ||
+          node.arguments[0].value.indexOf('axios/') === 0)
+      ) {
+        context.report(node, 'Deprecated require of axios package');
+      }
+    },
+    ImportDeclaration: (node) => {
```

Comment:
very unlikely to happen but, does this cover dynamic imports? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import

Reply:
Added code to handle them. Had to bump the ES version to something that supports dynamic imports, left it at latest.

Reply:
Now it does :)